src/documents/practices/FacilitateInclusively.html.md.handlebars

```diff
@@ -15,7 +15,7 @@ Thinking beyond technical limitations and avoiding getting bogged down in techni
 ## Try
 * Use accessible communication tools.
 * Consider how to support remote participation.
-* Make sure you space is physically accessible.
+* Make sure your space is physically accessible.
 * Ensure that everyone has the necessary background information.
```

Comment:
Perhaps it makes more sense to change the tense of these verbs to present participle, so something like
```
Using...
Considering...
Making sure...
Ensuring...
```

Reply:
Yes, changing these would mean that we have to change all the sentences which would be approximately 100lines of change. Should I proceed?

Reply:
Okay! I would be creating a seperate PR for this.

Reply:
new pr for this - https://github.com/inclusive-design/guide.inclusivedesign.ca/pull/36
thank you.